internal/consensus/common_test.go

```diff
@@ -432,12 +432,9 @@ func subscribeToVoterBuffered(cs *State, addr []byte) <-chan cmtpubsub.Message {
 // -------------------------------------------------------------------------------
 // application
 
-func fetchAppInfo(t *testing.T, app abci.Application) (*abci.InfoResponse, *mempl.LanesInfo) {
-	t.Helper()
-	resp, err := app.Info(context.Background(), proxy.InfoRequest)
-	require.NoError(t, err)
-	lanesInfo, err := mempl.BuildLanesInfo(resp.LanePriorities, resp.DefaultLane)
-	require.NoError(t, err)
+func fetchAppInfo(app abci.Application) (*abci.InfoResponse, *mempl.LanesInfo) {
+	resp, _ := app.Info(context.Background(), proxy.InfoRequest)
+	lanesInfo, _ := mempl.BuildLanesInfo(resp.LanePriorities, resp.DefaultLane)
```

Comment:
why skip the error checking here?

Reply:
Because functions `app.Info` and `BuildLanesInfo` are tested elsewhere. `fetchAppInfo` is an auxiliary function for consensus tests, and it would require a lot of changes just to test here that those functions do not return an error.